frontend/app/routes/_gcweb-app.personal-information.home-address.suggested.tsx

```diff
@@ -0,0 +1,154 @@
+import type { ReactNode } from 'react';
+
+import { type ActionFunctionArgs, type LoaderFunctionArgs, json, redirect } from '@remix-run/node';
+import { Form, Link, useLoaderData } from '@remix-run/react';
+
+import clsx from 'clsx';
+import { useTranslation } from 'react-i18next';
+
+import { Address } from '~/components/address';
+import { InputRadios } from '~/components/input-radios';
+import { getAddressService } from '~/services/address-service.server';
+import { sessionService } from '~/services/session-service.server';
+import { userService } from '~/services/user-service.server';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('personal-information');
+
+export const handle = {
+  breadcrumbs: [
+    { labelI18nKey: 'personal-information:home-address.suggested.breadcrumbs.home', to: '/' },
+    { labelI18nKey: 'personal-information:home-address.suggested.breadcrumbs.personal-information', to: '/personal-information' },
+    { labelI18nKey: 'personal-information:home-address.suggested.breadcrumbs.suggested-address' },
+  ],
+  i18nNamespaces,
+  pageIdentifier: 'CDCP-0008',
+  pageTitleI18nKey: 'personal-information:address.suggested.page-title',
+};
+
+/**
+ *
+ * loader, runs before rendering html code
+ *
+ */
+export async function loader({ request }: LoaderFunctionArgs) {
+  const userId = await userService.getUserId();
+  const userInfo = await userService.getUserInfo(userId);
+  const homeAddressInfo = await getAddressService().getAddressInfo(userId, userInfo?.homeAddress ?? '');
+  /**
+   * TODO
+   * CHANGE THE SOURCE OF THE SUGGESTED ADDRESS TO WHAT WS ADDRESS SERVICE IS RETURNING INSTEAD OF MAILING ADDRESS
+   */
+  const suggestedAddressInfo = await getAddressService().getAddressInfo(userId, userInfo?.mailingAddress ?? '');
+  const session = await sessionService.getSession(request.headers.get('Cookie'));
+  session.set('homeAddress', homeAddressInfo);
+  session.set('suggestedAddress', suggestedAddressInfo);
+
+  console.debug('LOADER SESSION home address :: ' + JSON.stringify(session.get('homeAddress'), null, 2));
+
+  return json({ homeAddressInfo, suggestedAddressInfo });
+}
+
+/**
+ * action, save address, either the entered address or the suggested address depending on the radio button choosen...
+ */
+export async function action({ request }: ActionFunctionArgs) {
+  const userId = await userService.getUserId();
+  const userInfo = await userService.getUserInfo(userId);
+  const homeAddressInfo = await getAddressService().getAddressInfo(userId, userInfo?.homeAddress ?? '');
+  console.debug('ACTION SESSION home address :: ' + JSON.stringify(homeAddressInfo, null, 2));
```

Comment:
These multiple occurrences of `console.debug(..)` can be removed.

Reply:
(or use `log.info(..)` if you feel this is important enough to help troubleshoot issues)

Reply:
Keep in mind that we only configure the logger on the server.. so you can use it in actions and loaders, but not the actual react component.

Reply:
console.debug occurrences have been removed